spreadsheet.py

```diff
@@ -34,10 +34,8 @@
 
 
 class Spreadsheet(Resource):
-    """ A wrapper around the Google API spreadsheet resource
-    
-    Constructs a spreadsheet API instance and creates user-friendly 
-    attributes and functions.  
+    """ A wrapper around the Google API spreadsheet resource constructs a
```

Comment:
After the change this is no longer a properly formatted Python docstring.  So we actually do want the short first line, and then more explanation on a different line.

The format for docstrings is described here: 
https://www.python.org/dev/peps/pep-0257/